luigi/file.py

```diff
@@ -45,6 +47,7 @@ def generate_tmp_path(self, path):
 
 
 class LocalFileSystem(FileSystem):
+
```

Comment:
Pylint claims they are.

From pep8: https://www.python.org/dev/peps/pep-0008/#blank-lines
"Method definitions inside a class are surrounded by a single blank line."

I guess it's up to interpretation if you should consider the constructor to be a class method or not. Pylint seems to believe that is the case.

Thanks for the code review.